buildSrc/src/main/groovy/org/apache/fineract/gradle/service/EmailService.groovy

```diff
@@ -43,12 +43,14 @@ class EmailService {
         this.properties.put("mail.smtp.auth", "false")
         this.properties.put("mail.smtp.starttls.enable", config.tls.toString())
         if(config.ssl) {
-            this.properties.put("mail.smtp.port", "465")
+            this.properties.put("mail.smtp.`port", "465")
```

Comment:
@tabrez-fiter there is an apostrophe before port which does not seem to be correct

Reply:
@faheem205 thanks' i already change